solutions/usaco-832.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: usaco-832
+source: USACO Bronze 2018 Open
+title: Milking Order
+author: Ananth Kashyap
+---
+
+## Solution 1
+
+[Official Analysis](http://www.usaco.org/current/data/sol_milkorder_bronze_open18.html)
+
+## Solution 2
+
+In this problem, there are three cases that we have to consider: 
+- The first of these cases is when cow one is fixed. In this case, the solution is simple, as we just output its fixed position. 
```

Comment:
```suggestion
- When the first cow is fixed, we output its position. 
```

Reply:
"When cow $1$" would be better imo

Reply:
^
Or even "Cow $1$ is fixed: blah blah blah"

Reply:
Also possibly have ordered list?
```
1. blah blah blah
2. ...
3. ...
```

Unordered is ok tho.